projects/RabbitMQ.Client/client/impl/ContentHeaderPropertyReader.cs

```diff
@@ -45,26 +45,28 @@
 
 namespace RabbitMQ.Client.Impl
 {
-    internal struct ContentHeaderPropertyReader
+    internal ref struct ContentHeaderPropertyReader
```

Comment:
It seems like

- `ReadBit`
- `ReadLong`
- `ReadLonglong`
- `ReadLongstr`
- `ReadShort`

are no longer used. Should we ditch them? Or is it worth keeping them around just in case?

Reply:
I'm fine with ditching them, I left them as they were there before.

Reply:
@michaelklishin @lukebakken any thoughts? Do you prefer to have them around?

Reply:
If code is unused and removing it does not change the API, remove it!